src/Raven.Server/RavenServer.cs

```diff
@@ -344,6 +345,38 @@ void ConfigureKestrel(KestrelServerOptions options)
             }
         }
 
+        private void UpdateCertificateExpirationAlert(object sender = null, EventArgs args = null)
+        {
+            var remainingDays = (Certificate.Certificate.NotAfter - Time.GetUtcNow().ToLocalTime()).TotalDays;
+            if (remainingDays <= 20)
```

Comment:
This can be negative? We want to set it to 0 then?

Reply:
Yes, I will display a relevant message if it is already expired.